src/cosalib/utils.py

```diff
@@ -0,0 +1,19 @@
+from subprocess import check_call, check_output, CalledProcessError
+
+
+def run_cmd(command, output=None):
+    '''
+    Run the given command using check_call/check_output and verify its return code.
+    @param str command command to be executed
+    '''
+    if output:
+        try:
+            output = check_output(command.split())
```

Comment:
We can just `return check_output(...)` here right?  

(It's a bit confusing that we're assigning to an input parameter value too)

Reply:
It definitely waits for the command; it's only the lower level `subprocess.Popen` that doesn't.

Reply:
A nice to know. Let me remove the check_call so.